webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -531,19 +319,18 @@ export const GlideTable: React.FC<GlideTableProps> = ({
         }, // TODO correctly handle error state
       });
     },
-    [appTheme, data, columnIds, columnDefs, getThemeVar],
+    [appTheme, data, columns, getThemeVar],
   );
 
   const onCellClicked: DataEditorProps['onCellClicked'] = useCallback(
     (cell: Item, event: CellClickedEventArgs) => {
       const [col, row] = cell;
 
       Loadable.forEach(data[row], (rowData) => {
-        const columnId = columnIds[col];
-        const cell = columnDefs[columnId].renderer(rowData, row);
+        const cell = columns[col].renderer(rowData, row);
 
         if (isLinkCell(cell)) {
-          handlePath(event as unknown as AnyMouseEvent, { path: cell.data.link.href });
+          onLinkClick?.(cell.data.link.href);
```

Comment:
consider having the gridCell expose an `onClick` prop that's called here with the click event if present, and moving the `handlePath` stuff to the column definition. it seems like this works because you're relying on `window.event`, which is deprecated.

Reply:
not sure passing the event works, because glide's `onCellClicked` only provides an event with the type `CellClickedEventArgs`, which is incompatible with the `AnyMouseEvent` that `handlePath` expects. I think the appropriate solution here is probably just to use the `useNavigate` hook.

Reply:
This worked previously and the underlying library didn't change, so it should still work.

Reply:
ah, misunderstood what the issue was, will go ahead and put together the solution you suggested